src/main.rs

```diff
@@ -57,7 +57,8 @@ async fn main() -> anyhow::Result<()> {
             .unwrap();
     });
 
-    Controller::new(pods, watcher::Config::default().labels("nais.io/naisjob=true"))
+    let label_env = env::var("WATCH_SELECTOR").unwrap_or("nais.io/naisjob=true".to_string());
```

Comment:
I think I would prefer that this line happens earlier (almost at the start), just to keep "configuration" on more or less the same place.

Reply:
Fair! On it :)